go.mod

```diff
@@ -38,6 +38,7 @@ require (
 	github.com/imdario/mergo v0.3.6 // indirect
 	github.com/jessevdk/go-flags v1.4.0
 	github.com/kr/text v0.2.0 // indirect
+	github.com/labstack/echo v3.3.10+incompatible
```

Comment:
please make sure only v4 is checked in
https://github.com/labstack/echo#example